src/state/remote_test.go

```diff
@@ -406,15 +439,19 @@ func complexRemoteExpectedConfigsRows() []*model.ConfigRowState {
 	}
 }
 
-func loadRemoteState(t *testing.T, projectStateFile string) *State {
-	api, _ := remote.TestStorageApiWithToken(t)
-	remote.SetStateOfTestProject(t, api, projectStateFile)
-
+func createManifest(t *testing.T) *manifest.Manifest {
 	projectDir := t.TempDir()
-	m, err := manifest.NewManifest(1, "connection.keboola.com", projectDir, "bar")
+	metadataDir := filepath.Join(projectDir, manifest.MetadataDir)
+	m, err := manifest.NewManifest(1, "connection.keboola.com", projectDir, metadataDir)
 	if err != nil {
 		assert.FailNow(t, err.Error())
 	}
+	return m
+}
+
+func loadRemoteState(t *testing.T, m *manifest.Manifest, projectStateFile string) *State {
+	api, _ := remote.TestStorageApiWithToken(t)
```

Comment:
ten `loadRemoteState` predpoklada nejaku fixnu strukturu projektu? tj ze to ma pomenovane branche configy atd..Ide mi oto ze ak to je jeden konkretny projekt tak si ho lokalne budem musiet napojit tiez, teraz tam mam nejaky svoj, ale velmi to na testovanie nevyuzivam

Reply:
ahoj. vsetko si test generuje, mal by sa dat pouzit lubovolny prazdny projekt. vzdy pred testom sa sice vsetko maze, ale tebe tam vtedy nieco neslo. 
jeden projekt je definovany v CI
a mam este jeden na lokalne testovanie, ... oba som vytvoril standardne cez UI, a nebol s nimi problem.
ak mas nejaky konkretny problem, tak mozeme pozriet.